Examples/Examples/FormViewExampleView.swift

```diff
@@ -96,18 +96,16 @@ extension FormViewExampleView {
     /// - Returns: The first identified feature.
     func identifyFeature(with proxy: MapViewProxy) async -> ArcGISFeature? {
         if let screenPoint = identifyScreenPoint,
-           let feature = try? await Result(awaiting: {
-               try await proxy.identify(
-                on: map.operationalLayers.first!,
+           let identifyLayerResult = try? await Result(awaiting: {
+               try await proxy.identifyLayers(
                 screenPoint: screenPoint,
                 tolerance: 10
                )
            })
             .cancellationToNil()?
             .get()
-            .geoElements
-            .first as? ArcGISFeature {
-            return feature
+            .first {
+            return identifyLayerResult.geoElements.first as? ArcGISFeature
```

Comment:
This will only return the first layer result that has `ArcGISFeature`s. We want the first layer result that has a feature form definition.
```suggestion
            .first(where: { result in
                if let feature = result.geoElements.first as? ArcGISFeature,
                   let _ = (feature.table?.layer as? FeatureLayer)?.featureFormDefinition {
                    return true
                } else {
                    return false
                }
            }) {
            return identifyLayerResult.geoElements.first as? ArcGISFeature
```